Casks/i/impel.rb

```diff
@@ -0,0 +1,32 @@
+cask "impel" do
+  version "1.0.3"
+  sha256 :no_check
+
+  url "https://impel-sparkle-updater.fly.dev/dmg/impel_current.dmg", verified: "impel-sparkle-updater.fly.dev/dmg/impel_current.dmg"
+  name "Impel"
+  desc "Proactive AI assistent for the future"
+  homepage "https://www.tryimpel.com/"
+
+  livecheck do
+    url "https://impel-sparkle-updater.fly.dev/appcast.xml"
+    strategy :sparkle do |item|
+      item.version.to_s
+    end
```

Comment:
Can pull the version directly

```suggestion
    strategy :sparkle, &:version
```

Reply:
Yep!